infrastructure/terraform/components/reporting/data_iam_policy_document_s3_backup.tf

```diff
@@ -0,0 +1,66 @@
+data "aws_iam_policy_document" "s3_backup" {
+  count = var.enable_s3_backup ? 1 : 0
+
+  statement {
+    actions = [
+      "s3:GetObject",
+      "s3:ListBucket",
+      "s3:ListBucketVersions",
+      "s3:GetBucketVersioning",
+      "s3:GetBucketTagging",
+      "s3:GetBucketLocation",
+      "s3:GetBucketNotification",
+      "s3:PutBucketNotification"
+    ]
+    resources = [
+      aws_s3_bucket.data.arn,
+      "${aws_s3_bucket.data.arn}/*",
+      aws_s3_bucket.results.arn,
+      "${aws_s3_bucket.results.arn}/*"
```

Comment:
Only need to backup the data bucket. Results bucket is only transient data that doesn't need to be backed up.